sklearn/metrics/_dist_metrics.pyx

```diff
@@ -119,9 +119,12 @@ cdef class DistanceMetric:
     "mahalanobis"   MahalanobisDistance   V or VI   ``sqrt((x - y)' V^-1 (x - y))``
     ==============  ====================  ========  ===============================
 
-    Note that "minkowski" with a non-None `w` parameter actually calls
-    `WMinkowskiDistance` with `w=w ** (1/p)` in order to be consistent with the
-    parametrization of scipy 1.8 and later.
+    .. deprecated:: 1.1
+        `WMinkowskiDistance` is deprecated in version 1.1 and will be removed in version 1.3.
+        Use `MinkowskiDistance` instead. Note that in `MinkowskiDistance`, the weights are
+        applied to the absolute differences already raised to the p power. This is different from
+        `WMinkowskiDistance` where weights are applied to the absolute differences before raising
+        to the p power. The deprecation aims to remain consistent with SciPy 1.8 convention.
```

Comment:
:+1: 
We must make sure this information is visible and understandable enough.
I think it's fine here but should the associated warning be extend to include all the explanation?

Reply:
That sounds good. Thank you!

Reply:
Maybe we could add a link to the documentation that includes the math formulas:

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.DistanceMetric.html